atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServices.java

```diff
@@ -48,7 +48,7 @@
 import com.palantir.util.Pair;
 
 public final class CassandraKeyValueServices {
-    private static final Logger log = LoggerFactory.getLogger(CassandraKeyValueService.class); // did this on purpose
+    private static final Logger log = LoggerFactory.getLogger(CassandraKeyValueServices.class); // did this on purpose
```

Comment:
The `// did this on purpose` comment seems to indicate we shouldn't blindly change the logger name. I think the expectation is that enabling logging for `com.palantir.atlasdb.keyvalue.cassandra.CassandraKeyValueService` should catch this as well since a lot of stuff in there relies on these utilities. One might argue to just enable logging on `com.palantir.atlasdb.keyvalue.cassandra` though that will catch a bunch of other stuff.